frontend/javascripts/admin/statistic/project_and_annotation_type_dropdown.tsx

```diff
@@ -108,6 +134,7 @@ function ProjectAndAnnotationTypeDropdown({
       options={filterOptions}
       optionFilterProp="label"
       value={selectedFilters}
+      popupMatchSelectWidth={false}
```

Comment:
This prop enables fully showing long project names as stated in the issue. 

But the antd docs to this prop state:
> Determine whether the popup menu and the select input are the same width. Default set min-width same as input. Will ignore when value less than select width. false will disable virtual scroll.

Where the last sentence is important IMO -> virtual scrolling is now disabled by this change. This means that for an organization with many projects, that the list might be slow.

Do you think this is acceptable?

Reply:
> An alternative could be to set a reasonable width (somewhat larger than before) and then show the full project name on hover. No real preference on my side.

That's definitely also doable :)

I'll switch to this behavior :+1: